planetiler-custommap/src/main/resources/samples/natural_earth.yml

```diff
@@ -0,0 +1,67 @@
+schema_name: Natural Earth
+schema_description: Natural Earth Oceans and Lakes example
+args:
+  minzoom: 0
+  maxzoom: 5
+
+sources:
+  natural_earth:
+    type: geopackage
+    url: "https://naciscdn.org/naturalearth/packages/natural_earth_vector.gpkg.zip"
+    projection: EPSG:4326
+
+layers:
+
+- id: ne_water_bodies
+  features:
+  - source: natural_earth
+    include_when:
+      - '${ feature.source_layer == "ne_10m_ocean" }'
+      - '${ feature.source_layer == "ne_10m_lakes" }'
+      - '${ feature.source_layer == "ne_50m_ocean" }'
+      - '${ feature.source_layer == "ne_50m_lakes" }'
+      - '${ feature.source_layer == "ne_110m_ocean" }'
+      - '${ feature.source_layer == "ne_110m_lakes" }'
```

Comment:
Minor: I think with #1063 you should now also be able to do this here and for class attribute:

```yaml
include_when:
${ feature.source_layer }:
- "ne_10m_ocean"
- "ne_10m_lakes"
- "ne_50m_ocean"
- "ne_50m_lakes"
- "ne_110m_ocean"
- "ne_110m_lakes"
```

Reply:
Thanks!
That seems to be applicable also to `min_zoom` and `max_zoom`